SETUP.md

```diff
@@ -336,3 +337,17 @@ docker run -p 8888:8888 -d recommenders:cpu
 ```
 
 You can then open the Jupyter notebook server at http://localhost:8888
+
+## Setup guide for making a release
+
+The process of making a new release and publishing it to pypi is as follows:
+
+1. Make sure that the code in main passes all the tests (unit and nightly tests).
+2. Update the version number in [reco_utils.py/__init__.py](reco_utils.py/__init__.py)
```

Comment:
don't we need to modify the version from staging?

```suggestion
1. Start with staging branch (all unit and nightly tests should be green)
1. Update the version number in [reco_utils.py/__init__.py](reco_utils.py/__init__.py)
1. Push PR back to staging
1. Create PR to main
1. Wait for code to merge to main branch
1. Check out updated main branch
```

Reply:
I'll try to come up with something simple

Reply:
@gramhagen I made some changes in this part